### PR TITLE
Umcs 479/extend authorization account properties

### PR DESCRIPTION
**Pull request by Ryouzanpaku**
Moved `iban`, `bic`, `holder`, `descriptor` properties from `Charge` class to a trait and use it in `Charge` and `Authorization` class

### CODE REVIEW

File: examples/PaylaterInvoice/index.php [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            }
+        } else {
+            $('button[type="submit"]').attr('disabled', 'disabled');
+            validCustomer = false;
+        }
+    })
+
+    form.addEventListener('submit', function(event) {
+        event.preventDefault();
+        let PaylaterInvoicePromise = paylaterInvoice.createResource();
```
Comment (mghazzouz) [inline examples/PaylaterInvoice/index.php:115]: camelCase to be used for variable names (PaylaterInvoicePromise -> paylaterInvoicePromise)

File: examples/PaylaterInvoice/index.php [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+</form>
+
+<script>
+    // Create an Unzer instance with your public key
+    let unzerInstance = new unzer('<?php echo UNZER_PAPI_PUBLIC_KEY; ?>');
+
+    // Create an Invoice Secured instance
+    let paylaterInvoice = unzerInstance.PaylaterInvoice();
+    paylaterInvoice.create({
+        containerId: 'example-invoice-secured',
```
Comment (mghazzouz) [inline examples/PaylaterInvoice/index.php:68]: no more secured ;) should be example-unzer-invoice or example-invoice-paylater

File: examples/PaylaterInvoice/index.php [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+<body style="margin: 70px 70px 0;">
+
+<p><a href="https://docs.unzer.com/reference/test-data" target="_blank">Click here to open our test data in new tab.</a></p>
+
+<form id="payment-form">
+    <div id="customer" class="field">
+        <!-- The customer form UI element will be inserted here -->
+    </div>
+    <div id="example-invoice-secured"></div>
```
[the remark this answers is not in the document]
Comment (Ryouzanpaku) [reply examples/PaylaterInvoice/index.php:56]: fixe in PR #80

File: examples/PaylaterInvoice/index.php (hunk 1 above)
Comment (Ryouzanpaku) [reply examples/PaylaterInvoice/index.php:115]: fixed in PR #80

File: examples/PaylaterInvoice/index.php (hunk 2 above)
Comment (Ryouzanpaku) [reply examples/PaylaterInvoice/index.php:68]: fixed in PR #80

File: CHANGELOG.md [hunk 4]
```diff
@@ -24,6 +24,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 * Remove redundant `currency` parameter from `Unzer::chargePayment()` method.
 * Add "geoLocation" property to all payment type classes.
 * Several minor improvements.
+* Add account information coming from PAPI to Authorize class.
```
Comment (mghazzouz) [inline CHANGELOG.md:27]: Should it be: to Authorize/Charge classes ?

Comment (Ryouzanpaku) [reply CHANGELOG.md:27]: No, only authorize, since the Charge class already had those properties.

Comment (mghazzouz) [reply CHANGELOG.md:27]: Ok I see, they were just extracted in the new class

**Review by mghazzouz — APPROVED** [review]
LGTM

**Pull request merged**